bin/auto-cpufreq

```diff
@@ -226,9 +226,22 @@ def main(config, daemon, debug, update, install, remove, live, log, monitor, sta
                 remove_complete_msg()
         elif update:
             root_check()
+            custom_dir = "/opt/auto-cpufreq/source"
+            for arg in sys.argv:
+                if arg.startswith("--update="):
+                    custom_dir = arg.split("=")[1]
+                    sys.argv.remove(arg)
+                    
+            if "--update" in sys.argv:
```

Comment:
I guess we don't need this part now if we'll use `custom_dir` with set location for now